apps/vaccitron/src/assets/demo-data.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
When is this demo data used?

Reply:
In dev and tests. Try to run the app locally